local/src/core/ProxyVerifier.cc

```diff
@@ -2253,7 +2274,7 @@ Load_Replay_Directory(swoc::file::path const &path,
 
       // Lambda suitable to spawn in a thread to load files.
       auto load_wrapper = [&]() -> void {
-        int k;
+        int k = 0;
```

Comment:
Did you get warnings about these? At least in this case, this does nothing since `k` is initialized before it is used anyway.

Reply:
No warning. After fixing the uninitialized remote_addr_size variable to accept4, I skimmed through and initialized all uninitialized declarations that I noticed.